app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -54,14 +54,14 @@ export class ArticleCoverImage extends Component {
     const { uploadError, uploadErrorMessage } = this.state;
     const uploadLabel = mainImage ? 'Change' : 'Add a cover image';
     return (
-      <div data-testid="article-form__cover" className="crayons-article-form__cover" role="presentation">
+      <div className="crayons-article-form__cover" role="presentation">
         {mainImage && (
           <img
             src={mainImage}
             className="crayons-article-form__cover__image"
             width="250"
             height="105"
-            alt="Post cover image"
+            alt="Post cover"
```

Comment:
The word image is redundant. Screen-readers already announce `<img />` tags as an image.